src/components/Versions.vue

```diff
@@ -45,61 +61,80 @@
-            });
-            versions.push({
-                name: "Course Management",
-                version: courseManagementVersion,
-                link:
-                    "https://github.com/upb-uc4/University-Credits-4.0/blob/" +
-                    "course_" +
-                    courseManagementVersion +
-                    "/product_code/lagom/course_service/CHANGELOG.md",
-            });
-            versions.push({
-                name: "User Management",
-                version: userManagementVersion,
-                link:
-                    "https://github.com/upb-uc4/University-Credits-4.0/blob/" +
-                    "user_" +
-                    userManagementVersion +
-                    "/product_code/lagom/user_service/CHANGELOG.md",
-            });
-            versions.push({
-                name: "Hyperledger Management",
-                version: hyperledgerManagementVersion,
-                link:
-                    "https://github.com/upb-uc4/University-Credits-4.0/blob/" +
-                    "hyperledger_" +
-                    hyperledgerManagementVersion +
-                    "/product_code/lagom/hyperledger_service/CHANGELOG.md",
-            });
-            versions.push({
-                name: "Hyperledger Course Management",
-                version: hyperledgerCourseManagementVersion,
-                link:
-                    "https://github.com/upb-uc4/University-Credits-4.0/blob/" +
-                    "hlcourse_" +
-                    hyperledgerCourseManagementVersion +
-                    "/product_code/lagom/hl_course_service/CHANGELOG.md",
+            onBeforeMount(() => {
+                getVersions();
             });
 
-            emit("versions", versions);
+            async function getVersions() {
+                busy.value = true;
+                let store = useStore();
+
+                let frontEndVersion = "v" + process.env.VUE_APP_VERSION;
+
+                let authenticationManagementVersion = await AuthenticationManagement.getVersion();
+                let courseManagementVersion = await CourseManagement.getVersion();
+                let userManagementVersion = await UserManagement.getVersion();
+                let hyperledgerManagementVersion = await HyperledgerManagement.getVersion();
+                let hyperledgerCourseManagementVersion = await HyperledgerCourseManagement.getVersion();
+
+                versions.push({
+                    name: "Frontend",
+                    version: frontEndVersion,
+                    link: "https://github.com/upb-uc4/ui-web/blob/" + frontEndVersion + "/CHANGELOG.md",
+                });
+                versions.push({
+                    name: "Authentication Management",
+                    version: authenticationManagementVersion,
+                    link:
+                        "https://github.com/upb-uc4/University-Credits-4.0/blob/" +
+                        "authentication_" +
```

Comment:
kind of unrelated fix, but lagom actually changed the naming of tags from what I thought it would be
```suggestion
                        "authentication-" +
```